azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/error/ErrorHandler.kt

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.chat.error
+
+import com.azure.android.communication.ui.chat.configuration.ChatCompositeEventsHandler
+import com.azure.android.communication.ui.chat.error.ErrorCode.Companion.CHAT_LOCAL_PARTICIPANT_EVICTED
+import com.azure.android.communication.ui.chat.models.ChatCompositeErrorEvent
+import com.azure.android.communication.ui.chat.models.ChatCompositeEventCode
+import com.azure.android.communication.ui.chat.models.ChatCompositeEventCode.Companion.CHAT_EVICTED
+import com.azure.android.communication.ui.chat.redux.AppStore
+import com.azure.android.communication.ui.chat.redux.state.ErrorState
+import com.azure.android.communication.ui.chat.redux.state.ReduxState
+import com.azure.android.communication.ui.chat.utilities.CoroutineContextProvider
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.cancel
+import kotlinx.coroutines.launch
+
+internal class ErrorHandler(
```

Comment:
I feel like we should have an event handler for this since it's technically not an error per se?

Reply:
renamed to `EventHandler`